gce/Makefile

```diff
@@ -53,8 +53,11 @@ PROM_ETCD_HOST_ENDPOINTS := $(shell python -c 'print str(["$(PREFIX)-etcd-%02d:9
 # Figure out what OS we're on (OSX or Linux) and lowercase it
 OS := $(shell uname -s | tr A-Z a-z)
 
+# get current directory
+CURRENT_DIR := $(shell pwd)
+
 # Add on the etcd nodes and master
-NODE_NUMBERS += $(ETCD_NODE_SUFFIXES) master
+NODE_NUMBERS += $(ETCD_NODE_SUFFIXES) master prom
```

Comment:
In the diags collection.  Without it, we won't collect diags from the prometheus node.